CHANGELOG.md

```diff
@@ -0,0 +1,101 @@
+# CHANGELOG
+
+Release Versions:
+- [1.0.0](#100)
+
+
+## Upcoming changes (in development)
+
+Version X.X.X introduces usage demos for CMake and ROS/ROS2 projects,
+a number of new helpful methods for the `robot_model::Model` class, and 
+a new type of `VelocityImpedance` controller.
+
+Many additional fixes and improvements have been made across the modules.
+
+The major version increment is due to some breaking changes in the `robot_model` API.
+
+### Breaking changes
+
+**robot_model**
+
+Breaking API changes include the renaming of some
+functions in the namespace `robot_model::Model` in an effort
+to improve consistency and reduce confusion. 
+
+The following functions have been renamed:
+- `forward_geometry()` -> `forward_kinematics()`
+- `forward_kinematic()` -> `forward_velocity()`
+- `inverse_kinematic()` -> `inverse_velocity()`
+
+
+### Features
+
+**demos**
+
+A top-level `demos` directory has been added to contain various usage examples
+and demonstrations for the control libraries.
+- Add CMake control loop examples with robot kinematics and task-space control (#96, #97)
+- Add ROS demo package (#115)
+- Add ROS2 demo package (#117)
+
+**state_representation**
+- Implement Jacobian operations with other matrices and Jacobian objects (#92)
+
+**robot_model**
+- Add an inverse kinematics function to calculate joint positions for a cartesian position of a robot model (#46)
+- Add a function to check if a joint state is in range for a given robot model (#91)
+- Add a getter function to retrieve the pinocchio model of a given robot model (#111)
+- Add a function to calculate the Jacobian time derivative (#118)
+
+**controllers**
+- Add a velocity impedance controller (#94)
+
+### Fixes and improvements
+- Fix Ring DS rotation offset and boundary condition (#90)
+- Correct copy constructor from derived Cartesian classes (#100)
+- Correct copy constructor from derived Joint classes (#101)
+- Fix Quaternion distance calculation (#105)
+- Correct joint names potential mismatch in inverse kinematics (#106)
+- Do not run the demos container if building fails (#107)
+- fix: remove the non defined operators that were cause linking issues (#108)
+- Revise install script (#89, #121)
+
+### Behind the scenes
+- Improve kinematics test coverage (#86)
+- Fix indentations issues and minor code duplication (#93)
+- CI job to build and test in Release configuration (#87)
+- Delete inaccessible getters and setters (#102)
```

Comment:
Would you consider those this and the next one to be fixes and improvements? They go together with 100 and 101 for me.